aci/convert/convert.go

```diff
@@ -181,9 +181,12 @@ func (s serviceConfigAciHelper) getAciContainer() (containerinstance.Container,
 	if err != nil {
 		return containerinstance.Container{}, err
 	}
-
+	containerName := s.Name
```

Comment:
I don't think we can easily support container_name in ACI, as we use the _ naming convention to retrieve containers from a compose stack. This would break other scenarios like `docker ps` => `docker logs` etc.

Reply:
in local docker-compose, container_name means container has no `projectname_` prefix. I don't think we should support it in ACI with a different meaning, that would be confusing more than anything else. We could just ignore it with a warning